### PR TITLE
feat: extended error message with support links

### PR DESCRIPTION
**Pull request by anandgupta42**
## Overview

Add slack channel and github issues link to error messages
## Checklist

- [X] I have run this code and it appears to resolve the stated issue
- [X] `README.md` updated and added information about my change

### CODE REVIEW

**Review by mdesmet — COMMENTED** [review]
Please add punctuation in every error.

File: src/document_formatting_edit_provider/dbtDocumentFormattingEditProvider.ts [hunk 1]
```diff
@@ -70,15 +70,18 @@ export class DbtDocumentFormattingEditProvider
           );
           window.showErrorMessage(
             "Could not process difference output from sqlfmt. Detailed error: " +
-              error,
+              error +
+              " If the issue persists, please report it on GitHub (https://bit.ly/dpuissue) or seek help in our dbt Community Slack channel #tools-dbt-power-user (https://bit.ly/dpuslack)",
```
Comment (mdesmet) [inline src/document_formatting_edit_provider/dbtDocumentFormattingEditProvider.ts:74]: Should also utils method

Comment (mdesmet) [inline src/document_formatting_edit_provider/dbtDocumentFormattingEditProvider.ts:74]: Could also use the utils method.

File: src/utils.ts [hunk 2]
```diff
@@ -115,3 +115,10 @@ export function substituteSettingsVariables(value: string): string {
   );
   return value;
 }
+
+export function extendErrorWithSupportLinks(error: string): string {
+  return (
+    error +
+    " If the issue persists, please report it on GitHub (https://bit.ly/dpuissue) or seek help in our dbt Community Slack channel #tools-dbt-power-user (https://bit.ly/dpuslack)"
```
Comment (mdesmet) [inline src/utils.ts:122]: Check if these can be made into links.

Comment (anandgupta42) [reply src/utils.ts:122]: I couldn't find a way to do that.

Comment (anandgupta42) [reply src/utils.ts]: made it markdown

File: src/manifest/dbtProject.ts [hunk 3]
```diff
@@ -223,14 +224,19 @@ export class DBTProject implements Disposable {
     } catch (exc: any) {
       if (exc instanceof PythonException) {
         window.showErrorMessage(
-          "An error occured while initializing the dbt project: " +
-            exc.exception.message,
+          extendErrorWithSupportLinks(
+            "An error occured while initializing the dbt project: " +
```
Comment (lionsardesai) [inline src/manifest/dbtProject.ts:228]: dot here as well?

File: src/dbt_client/index.ts [hunk 4]
```diff
@@ -198,7 +198,7 @@ export class DBTClient implements Disposable {
       message &&
       (versionCheck === "both" || versionCheck === "error message")
     ) {
-      window.showErrorMessage(message);
+      window.showErrorMessage(extendErrorWithSupportLinks(message));
```
Comment (lionsardesai) [inline src/dbt_client/index.ts:201]: dot here as well?

File: src/dbt_client/dbtCommandQueue.ts [hunk 5]
```diff
@@ -37,7 +37,9 @@ export class DBTCommandQueue {
             await command(token);
           } catch (error) {
             window.showErrorMessage(
-              `Could not run command '${statusMessage}': ` + error,
+              extendErrorWithSupportLinks(
+                `Could not run command '${statusMessage}': ` + error,
```
Comment (lionsardesai) [inline src/dbt_client/dbtCommandQueue.ts:41]: le dot

**Review by lionsardesai — APPROVED** [review]

**Pull request merged**